repose-aggregator/components/filters/header-translation-filter/src/main/scala/org/openrepose/filters/headertranslation/HeaderTranslationFilter.scala

```diff
@@ -101,3 +108,9 @@ class HeaderTranslationFilter @Inject()(configurationService: ConfigurationServi
     initialized = true
   }
 }
+
+object HeaderTranslationFilter {
+  private final val DefaultConfig = "header-translation.cfg.xml"
+  private final val SchemaFileName = "/META-INF/schema/config/header-translation.xsd"
+  private final val QualityReqex = ";q=\\d[^;]*".r
```

Comment:
Side note: I've always found that using single-quotes for regexes may lead to confusion, especially when you have to escape (or double escape, as it were) things. You might consider using triple quotes instead.

`""";q=\\d[^;]*""".r`